modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -547,4 +547,41 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-195725. Verify that it's possible to delete the MDF Claim while in draft status"
+	@priority = 4
+	test PMUUserCanDeleteMDFClaimInDraftStatus {
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given that the PMU user goes to the created MDF") {
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+
+			WaitForElementPresent(
+				key_campaignName = "Campaign Name",
+				locator1 = "PRMMDFRequest#REQUEST_ID");
```

Comment:
Hey @brunadmelo , is this `WaitForElementPresent` really necessary? If you open the MDF Requests Site Page the  Campaign Name should already be there.

Reply:
@guivilarim indeed this is not necessary, I will remove this assert. Was using that to run the test on my home connection which is more slow than the one from the Liferay office.